tests/sentry/runner/commands/test_backup.py

```diff
@@ -0,0 +1,33 @@
+import os
+
+from click.testing import CliRunner
+from django.utils.functional import cached_property
+
+from sentry.runner.commands.backup import export, import_
+from sentry.testutils import CliTestCase, TransactionTestCase
+
+
+class BackupTest(CliTestCase):
+    command = export
```

Comment:
It's goofy that we mix nomenclature between backup/restore and export/import. Ofc not in scope here to do anything about it, but wanted to start surfacing the thought.

Reply:
I agree!